### PR TITLE
[FE] LandingPage specs

### PR DESCRIPTION
**Pull request by shirve**
### Changes
- Added `LandingPage` specs
- Moved `setupTests` localStorage mock to the spec that needed it because it was causing problems in other specs
- Updated specs that stopped working after changing `profile` factory

### CODE REVIEW

File: team-fingerprints-web/src/setupTests.ts [hunk 1]
```diff
@@ -12,16 +12,6 @@ jest.mock("@auth0/auth0-react", () => {
   return mockAuth0;
 });
 
-// access to localStorage
```
Comment (DawidHaratyk) [inline team-fingerprints-web/src/setupTests.ts:15]: Can I know why did you remove these localStorage lines? Some files are managing local storage

**Review by DawidHaratyk — CHANGES_REQUESTED** [review]

File: team-fingerprints-web/src/setupTests.ts [hunk 2]
```diff
@@ -12,16 +12,6 @@ jest.mock("@auth0/auth0-react", () => {
...
-Object.defineProperty(window, "localStorage", {
-  value: {
-    setItem: jest.fn(),
-    getItem: jest.fn(),
-    removeItem: jest.fn(),
-  },
-  writable: true,
-});
-
```
Comment (DawidHaratyk) [inline team-fingerprints-web/src/setupTests.ts:24]: Can I know why did you remove these localStorage lines? Some files are managing local storage

**Review by DawidHaratyk — CHANGES_REQUESTED** [review]

File: team-fingerprints-web/src/setupTests.ts (hunk 2 above)
Comment (shirve) [reply team-fingerprints-web/src/setupTests.ts:24]: It got moved to the spec that was using it as it was causing problems with mocking values into localStorage elsewhere

**Comment by tobob** [comment]
@shirve merge conflicts ;)

**Pull request merged**